mod/item.php

```diff
@@ -496,7 +491,7 @@ function item_post(App $a) {
 				continue;
 			}
 
-			$success = handle_tag($a, $body, $inform, $str_tags, (local_user()) ? local_user() : $profile_uid , $tag, $network);
+			$success = handle_tag($a, $body, $inform, $str_tags, local_user() ? local_user() : $profile_uid, $tag, $network);
```

Comment:
You can use `local_user() ? : $profile_uid` or `defaults(local_user(), $profile_uid)` as well here to avoid the repetition in the ternary operation.